README.md

```diff
@@ -478,6 +478,7 @@ API | Description | Auth | HTTPS | CORS |
 | [PostcodeData.nl](http://api.postcodedata.nl/v1/postcode/?postcode=1211EP&streetnumber=60&ref=domeinnaam.nl&type=json) | Provide geolocation data based on postcode for Dutch addresses | No | No | Unknown |
 | [Postcodes.io](https://postcodes.io) | Postcode lookup & Geolocation for the UK | No | Yes | Yes |
 | [REST Countries](https://restcountries.eu) | Get information about countries via a RESTful API | No | Yes | Unknown |
+| [SmartIP.io](https://smartip.io) | IP Geolocation and Threat Intelligence API offering 250,000 requests per month for free | `apiKey` | Yes | Yes |
```

Comment:
Thank you!

Please remove the free tier info:
```suggestion
| [SmartIP.io](https://smartip.io) | IP Geolocation and Threat Intelligence | `apiKey` | Yes | Yes |
```

Reply:
PR from (smartip-io:patch-2) updated as requested.